empress/support_files/templates/side-panel.html

```diff
@@ -7,15 +7,15 @@
     <input type="text" id="quick-search" placeholder="Search by node name..."
            style="flex-grow: 1;" autocomplete="off" class="empress-input">
     <button id="side-header-search-btn" style="flex-grow: 1;">Search</button>
-    <button id="hide-ctrl" title="Hide control panel"
+    <button id="hide-ctrl" title="Hide empress-control panel"
```

Comment:
```suggestion
    <button id="hide-ctrl" title="Hide control panel"
```

Since this is a message the user will see.